packages/app/cypress/e2e/reporter_header.cy.ts

```diff
@@ -0,0 +1,84 @@
+describe('Reporter Header', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer()
+    cy.visitApp()
+    cy.contains('dom-content.spec').click()
+  })
+
+  context('Specs Shortcut', () => {
+    it('displays the currently running spec as highlighted in the Specs List', () => {
```

Comment:
This doesn't appear to to be verify the spec is highlighted, just has the selected data attribute. Maybe the name is mis-leading? Or are there additional check missing?

Reply:
I updated the CT test for this and removed it from the E2E section of the spreadsheet